k2/csrc/fsa_util.cc

```diff
@@ -102,6 +105,15 @@ void SplitStringToVector(const std::string &in, const char *delim,
   }
 }
 
+struct PairHash {
```

Comment:
there is a template `PairHash` in `until.h`, just using that.

Reply:
mm, in this case I just want to take the 1st element as it contains the same information as the 2nd, just differently hashed.  It's faster.

Reply:
Done.